### ISSUE TITLE
[Bug]: PlayCover Genshin Impact "Login Expired, Log In Again"

### ISSUE THREAD

**Issue by Marzipanini** [PlayCover/PlayCover#1053]
### Describe the bug

I've been having trouble with logging in, so I followed the documentation to be able to log in. However, once I open Genshin Impact from PlayCover once CSRUTIL is re-enabled, Genshin says that my login expired and that I have to log in again. I have tried reproducing the steps just in case I did something wrong to no avail.

### Steps to reproduce

1. Restarting MacBook to get to startup options.
2. Use Terminal to csrutil disable. 
3. Restart MacBook. 
4. Use Terminal and the command: sudo nvram boot-args="amfi_get_out_of_my_way=0x1 ipc_control_port_options=0"
5. Restart MacBook. 
6. Open Genshin Impact in PlayCover, login, quit before going through the door. 
7. Restart to access startup options. 
8. Use Terminal to use: csrutil enable.
9. Restart and open Genshin Impact with PlayCover.
10. Game opens with the dialogue, "Login Expired, please log in again"

### Expected behaviour

I expect to not have this error and be able to log in. 

### Crash log

```shell
No crash report. Note, I am using version 2.0.5 of PlayCover. It was not an option in the dropdown menu.
```

### What version of PlayCover are you using?

2.0.4

### What version of macOS are you using?

Ventura (macOS 13)

### Issue Language

- [X] Yes my issue is written in English

**Comment by MEMORIEmusic** [comment]
1. Make sure SIP is enabled
2. Use [PlayCover Beta 3.0](https://github.com/PlayCover/PlayCover/releases/tag/3.0.0-beta.1)

If after installing 3.0 PlayCover asks to update, click “Skip this version”.

You should be able to login(with SIP enabled) after installing 3.0

**Comment by Marzipanini** [comment]
Ok, thank you for the help! Turns out I was doing two things wrong. 

1. I was not using PlayCover 3.0
2. I was trying to Sign in with Apple. Once I added an email and password to my Genshin account, I was able to sign in on PlayCover without any issues. 

Solution: 
1. Delete old PlayCover app. 
2. Download PlayCover 3.0.
3. Login to Genshin with email and password instead of through Apple. 

Thank you everyone!
